Modules/IO/BioRad/src/itkBioRadImageIO.cxx

```diff
@@ -31,9 +31,6 @@
 #include "itkMakeUniqueForOverwrite.h"
 
 #define BIORAD_HEADER_LENGTH 76
-#define BIORAD_NOTE_LENGTH 96
```

Comment:
I would prefer to comment these out. While this code is unlikely to change, having this commented out is better than removed.

Reply:
For this file only, or all similar cases in this same commit?

Reply:
For all the changes in this commit.